core/tests/protractor_desktop/navigation.js

```diff
@@ -214,6 +215,13 @@ describe('Static Pages Tour', function() {
       by.css('.protractor-test-thanks-page')).isPresent()).toBe(true);
   });
 
+  it('visits the Welcome Guide Page', function() {
+    browser.get('/welcome_guide');
```

Comment:
@kmaraj Can we navigate to this link using the buttons a user would instead of going straight to the URL? See https://github.com/oppia/oppia/pull/9247#discussion_r420500615

Reply:
I made changes to these tests can you let me know if they are okay?